### PR TITLE
Rewrite to use Tower CLI

### PR DESCRIPTION
**Pull request by ewels**
Rewrite to use the new Tower command line tool: https://github.com/seqeralabs/tower-cli

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -30,19 +33,18 @@ inputs:
...
-    default: '[]'
     required: false
 
 runs:
   using: 'docker'
   image: Dockerfile
   env:
     TOWER_WORKSPACE_ID: ${{ inputs.workspace_id }}
-    TOWER_BEARER_TOKEN: ${{ inputs.bearer_token }}
+    TOWER_API_ENDPOINT: ${{ inputs.api_endpoint }}
```
Comment (ggabernet) [inline action.yml:47]: That's a nice addition, can use our private installation too :)

File: README.md [hunk 2]
```diff
@@ -50,7 +48,7 @@ jobs:
         # Use repository secrets for sensitive fields
         with:
           workspace_id: ${{ secrets.TOWER_WORKSPACE_ID }}
-          bearer_token: ${{ secrets.TOWER_BEARER_TOKEN }}
+          access_token: ${{ secrets.TOWER_ACCESS_TOKEN }}
           compute_env: ${{ secrets.TOWER_COMPUTE_ENV }}
```
Comment (ggabernet) [inline README.md:53]:
```suggestion
          api_endpoint: ${{ secrets.TOWER_API_ENDPOINT }}
          compute_env: ${{ secrets.TOWER_COMPUTE_ENV }}
```

Comment (ggabernet) [reply README.md:53]: Not sure if you also want to add the new param here...

**Review by ggabernet — APPROVED** [review]
Looks good to me :)

**Comment by ewels** [comment]
Ok, this is working now and I think is good for a 2.0 release. However, it contains breaking changes so won't fix existing pipelines.

Pipelines are currently fixed to `master`, so I think I will switch this repo to use `main` and then push a hacky backwards compatibility fix to `master`.

**Pull request merged**